shared/chat/conversation/input-area/normal/platform-input.native.tsx

```diff
@@ -110,6 +110,9 @@ class _PlatformInput extends React.PureComponent<PlatformInputPropsInternal, Sta
     const text = this.getText()
     if (text) {
       this.props.onSubmit(text)
+      if (this.state.expanded) {
+        this.expandInput()
```

Comment:
this function has an unfortunate name, how would you feel about `toggleExpandedInput` or something?